core/unit_test/TestViewAPI.hpp

```diff
@@ -1087,6 +1087,12 @@ class TestViewAPI {
     dView4_unmanaged unmanaged_dx = dx;
     ASSERT_EQ(dx.use_count(), 1);
 
+    // Test self assignment
+    dx = dx;
+    ASSERT_EQ(dx.use_count(), 1);
+    dx = reinterpret_cast<typename dView4::uniform_type &>(dx);
```

Comment:
This guy triggers the viewtracker.assign function. Note the type changes because View is a variadic template, and uniform_type spells out every argument.